File: Entidades/PatUsuEN.vb

Public Class PatUsuEN
    Private _codusu As Integer
    Public Property CodUsu() As Integer
        Get
            Return _codusu
        End Get
        Set(ByVal value As Integer)
            _codusu = value
        End Set
    End Property

    Private _codpat As Integer
    Public Property CodPat() As Integer
        Get
            Return _codpat
        End Get
        Set(ByVal value As Integer)
            _codpat = value
        End Set
    End Property

    Private _usuariopat As String
    Public Property UsuarioPat() As String
        Get
            Return _usuariopat
        End Get
        Set(ByVal value As String)
            _usuariopat = value
        End Set
    End Property

    Private _descpat As String
    Public Property DescPat() As String
        Get
            Return _descpat
        End Get
        Set(ByVal value As String)
            _descpat = value
        End Set
    End Property


    Private _activo As Boolean
    Public Property Activo() As Boolean
        Get
            Return _activo
        End Get
        Set(ByVal value As Boolean)
            _activo = value
        End Set
    End Property

    Private _dvh As Integer
    Public Property DVH() As Integer
        Get
            Return _dvh
        End Get
        Set(ByVal value As Integer)
            _dvh = value
        End Set
    End Property

End Class

File: Entidades/PatUsuEN.cs

namespace Entidades
{
    public class PatUsuEN
    {
        private int _codusu;

        public int CodUsu
        {
            get
            {
                return _codusu;
            }

            set
            {
                _codusu = value;
            }
        }

        private int _codpat;

        public int CodPat
        {
            get
            {
                return _codpat;
            }

            set
            {
                _codpat = value;
            }
        }

        private string _usuariopat;

        public string UsuarioPat
        {
            get
            {
                return _usuariopat;
            }

            set
            {
                _usuariopat = value;
            }
        }

        private string _descpat;

        public string DescPat
        {
            get
            {
                return _descpat;
            }

            set
            {
                _descpat = value;
            }
        }

        private bool _activo;

        public bool Activo
        {
            get
            {
                return _activo;
            }

            set
            {
                _activo = value;
            }
        }

        private int _dvh;

        public int DVH
        {
            get
            {
                return _dvh;
            }

            set
            {
                _dvh = value;
            }
        }
    }
}